### ISSUE TITLE
djpr_plot_server empty header tags

### ISSUE THREAD

**Issue by mrjoh3** [djpr-data/djprshiny#21]
the plot server seems to be resulting in empty header tags. This is causing one of the biggest accessibility issues for both the labour and trade dashboards.

**Comment by mrjoh3** [comment]
once fixed see issues https://github.com/djpr-data/djprtradedash/issues/108 and https://github.com/djpr-data/djprlabourdash/issues/149

**Comment by MattCowgill** [comment]
I feel bad for leaving this repo such a mess...

**Comment by mrjoh3** [comment]
> I feel bad for leaving this repo such a mess...

theres no need, it should be easy to fix and the issues only came up when we were doing accessibility assessments

**Comment by Henry-DJPR** [comment]
Fixed with djpr_async_server/ui #27
